js/privatebin.js

```diff
@@ -5558,6 +5624,16 @@ jQuery.PrivateBin = (function($, RawDeflate) {
             try {
                 Model.getPasteId();
             } catch (e) {
+
+                // check if protected pathname
+                const url = new URL(window.location);
+
+                // if protected request password
+                if(url.hash.startsWith('#protected/')) {
```

Comment:
I think this can be simplified to just:

```suggestion
                // if protected pathname, request password
                if(window.location.hash.startsWith('#protected/')) {
```